src/sentry/static/sentry/app/components/activity/item/index.jsx

```diff
@@ -0,0 +1,121 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import TimeSince from 'app/components/timeSince';
+import space from 'app/styles/space';
+import textStyles from 'app/styles/text';
+
+import ActivityAvatar from './avatar';
+import ActivityBubble from './bubble';
+
+class ActivityItem extends React.Component {
+  static propTypes = {
+    id: PropTypes.string,
+    date: PropTypes.string,
+    author: PropTypes.shape({
+      type: ActivityAvatar.propTypes.type,
+      user: ActivityAvatar.propTypes.user,
+    }),
```

Comment:
Can we mark all the required props here (and in any other applicable components). Since we might end up using this component in various it would be good to be more explicit about which parts are required vs optional

Reply:
Updated, everything is pretty much optional besides author